### PR TITLE
Addedd the pic feature to the build options.

### PR DESCRIPTION
**Pull request by ereichert**
I added the feature to tlx2 

features = ["build", "pic"]

It works perfectly.

### CODE REVIEW

**Comment by ereichert** [comment]
Hold off on this for a second.  I need to make sure the build works with zlib and pushed on the wrong branch.

**Comment by ereichert** [comment]
@ryandbair This works great.  Please review.

File: build.rs [hunk 1]
```diff
@@ -339,7 +340,10 @@ fn check_features(infos: &Vec<(&'static str, Option<&'static str>, &'static str)
 fn main() {
 	if env::var("CARGO_FEATURE_BUILD").is_ok() {
 		println!("cargo:rustc-link-search=native={}", search().join("lib").to_string_lossy());
-		println!("cargo:rustc-link-lib=z");
+
+		if env::var("CARGO_FEATURE_ZLIB").is_ok() && cfg!(target_os = "linux") {
```
Comment (ryandbair) [inline build.rs:344]: Is this really only applicable to Linux? I would think this would affect all platforms.

Comment (ereichert) [inline build.rs:344]: It does but the Windows dependency is handled in a much different way according to what I read on the subject.  Leaving open to any OS seemed to be more of a problem than restricting until someone verified it works on other OS.

**Pull request merged**